Dockerfile

```diff
@@ -0,0 +1,40 @@
+# syntax = docker/dockerfile:1
+FROM ruby:3.0-slim
+ENV BUNDLE_PATH="$GEM_HOME" \
+    BUNDLE_BIN="$GEM_HOME/bin" \
+    BUNDLE_SILENCE_ROOT_WARNING=1 \
+    BUNDLE_APP_CONFIG="$GEM_HOME" \
+    BUNDLE_JOBS=4 \
+    BUNDLE_RETRY=3 \
+    GEM_HOME=/usr/local/bundle
+ENV PATH="$BUNDLE_BIN:$PATH"
+ENV LANG=C.UTF-8
+
+RUN <<-EOT
+    set -xe
+    apt-get update
+    apt-get install -y build-essential git
+EOT
+
+WORKDIR /app
+
+COPY Gemfile* *.gemspec /app/
+COPY lib /app/lib
+COPY exe /app/exe
```

Comment:
This uses for `bundle install`.
After build, `/app` on container is mounted by base path via volume.